plutus-benchmark/bls12-381-costs/test/bls12-381-costs.golden

```diff
@@ -103,7 +103,7 @@ VRF example
 
     n     Script size             CPU usage               Memory usage
   ----------------------------------------------------------------------
-    -     783   (4.8%)      1547576705  (15.5%)          400191   (2.9%) 
```

Comment:
@kwxm any idea why dropping the fully applied condition for inlining functions in the PIR inliner could increase the script size? We do do checks that size shouldn't increase before inlining. So I wonder if we've missed something?

Reply:
Yeah, that sounds suspicious. Is there a golden test where the size increases?

Reply:
Not so far, or maybe I don't know how to find them.

Reply:
OK we found out this was just from `master`.